src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -549,11 +549,6 @@ internal Enumerator(LinkedList<T> list)
                 _index = 0;
             }
 
-            private Enumerator(SerializationInfo info, StreamingContext context)
-            {
-                throw new PlatformNotSupportedException();
-            }
-
```

Comment:
@ViktorHofer, this type isn't marked as [Serializable]. I assume the interfaces remained for public API compatibility, but there's no issue removing this private ctor, right?

Reply:
Yeah that can be safely removed :)